tests/acceptance/banner.feature

```diff
@@ -0,0 +1,52 @@
+Feature: Banner
+  In order to manage Banner article in the web
+  I need to create modify trash archived Check-In And publish and Unpublish Banner 
+
+  Background:
+    Given Joomla CMS is installed
```

Comment:
Thanks for the pull @dsmewara ^_^

This given needs to be removed as agreed at https://github.com/joomla-projects/gsoc16_browser-automated-tests/pull/54#discussion_r71760814.